index.html

```diff
@@ -6,12 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```

Comment:
- [ ] [REQUIRED] Please, replace the `Document` of the `<title>` tag with an appropriate name :recycle: to follow SEO best practices :receipt:.